packages/commerce-sdk-react/src/hooks/types.ts

```diff
@@ -226,3 +227,17 @@ export type TMutationVariables = {
     parameters?: {[key: string]: string | number | boolean | string[] | number[]}
     headers?: {[key: string]: string}
 } | void
+
+export type SDKClientTransformer<T> = (
+    params: T,
+    methodName: string,
+    options: any
+) => any | Promise<any>
```

Comment:
From what I see in the default transformer, it looks like the return object is like `options` but with some overrides/additions. If that's the case, can we make the type clearer?

```suggestion
export type SDKClientTransformer<T> = (
    params: T,
    methodName: string,
    options: SomeOptions
) => SomeOptions | Promise<SomeOptions>
```